docs/user-guide/manage-branches.md

```diff
@@ -1,6 +1,30 @@
 # Manage Branches
 
-This page describes how to manage branches in the created component, whether it is an application, library, autotest or infrastructure.
+This page describes how to manage branches in the created component, whether it is an application, library, autotest or infrastructure. It also briefly explains two approaches of managing custom pipelines for codebases.
+
+## Custom Pipeline Management
+
+There are two approaches to define a codebases with custom Tekton pipelines:
+
+  * **Using custom build tool and framework**
+  * **Using codebase branch settings**
+
+### Custom Build Tool/Framework Approach
+
+This approach involves [adding your application](../use-cases/tekton-custom-pipelines.md) with custom language type and framework.
+
+Follow this approach if you need to define multiple similar applications with identical settings, as this approach would be more beneficial.
+
+###  Codebase Branch Settings Add-Ons Approach
```

Comment:
we use add-ons approach for Custom Build Tool/Framework Approach approach too. This is about tekton resources storing but not about applying in KRCI.